mods/ra/rules/misc.yaml

```diff
@@ -297,7 +297,7 @@ powerproxy.parabombs:
 	AirstrikePower:
 		Icon: parabombs
 		Description: Parabombs (Single Use)
-		LongDesc: Three Badgers drops a load of parachuted\nbombs on your target.
+		LongDesc: A Badger drops a load of parachuted bombs on your target.
```

Comment:
IMO it would make more sense (in a followup PR to avoid scope creeping this one) to fix this to use three planes like the main power.

Reply:
I'm interested to make this change.